mtags/src/main/scala/scala/meta/internal/mtags/JavaMtags.scala

```diff
@@ -0,0 +1,172 @@
+package scala.meta.internal.mtags
+
+import com.thoughtworks.qdox._
+import com.thoughtworks.qdox.model.JavaClass
+import com.thoughtworks.qdox.model.JavaField
+import com.thoughtworks.qdox.model.JavaMember
+import com.thoughtworks.qdox.model.JavaMethod
+import com.thoughtworks.qdox.model.JavaModel
+import com.thoughtworks.qdox.parser.ParseException
+import java.io.StringReader
+import java.util.Comparator
+import scala.collection.mutable
+import scala.meta.inputs.Input
+import scala.meta.inputs.Position
+import scala.meta.internal.semanticdb.Language
+import scala.meta.internal.semanticdb.SymbolInformation.Kind
+import scala.meta.internal.semanticdb.SymbolInformation.Property
+import scala.meta.internal.mtags.Enrichments._
+
+/**
+ * Utility to generate method symbol  disambiguators according to SemanticDB spec.
```

Comment:
double space between `symbol` and `disambiguators`

Reply:
Removed, only if we had a formatting tool that could reformat our comments 🤔